examples/java/build.gradle

```diff
@@ -25,6 +25,7 @@ plugins {
 }
 
 applyJavaNature(
+  enableStrictDependencies: true,
```

Comment:
Please merge website-revamp into this branch (that will remove these duplicate changes to the examples/ directory).

Reply:
I don't know if it's possible if my branch is on different repository, I tried but without success

Reply:
Discussed this offline in #beam-website-finalize slack. I think you have a handle on this now, but let me know if you need help. It should be possible.

Reply:
Much better, thank you